### PR TITLE
updated user login and signup

### PR DESCRIPTION
**Pull request by Ranapriyanshi**
Updated components for user login and signup.

### CODE REVIEW

File: ornate_frontend/src/context/AuthContext.js [hunk 1]
```diff
@@ -18,6 +18,16 @@ export const AuthContextProvider = ({ children }) => {
     user: null,
   });
 
+  // Checking weather the user is there in the local storage and login the user again.
+
+  useEffect(() => {
+    const user = JSON.parse(localStorage.getItem("user"));
```
Comment (PatelNeelMahesh) [inline ornate_frontend/src/context/AuthContext.js:24]: Every JavaScript file that is loaded on your domain has access to Local Storage. If malicious JavaScript code is added by you or your dependencies, they can retrieve user data or tokens you use to authenticate with APIs.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Remove local storage for storing user details.

File: ornate_frontend/src/context/AuthContext.js (hunk 1 above)
Comment (Ranapriyanshi) [reply ornate_frontend/src/context/AuthContext.js:24]: Made the required commits.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**